LibSerialize.lua

```diff
@@ -1112,14 +1156,31 @@ LibSerialize._WriterTable = {
                 end
 
                 local mapCountWritten = 0
-                for k, v in pairs(tab) do
-                    -- Exclude keys that have already been written via the previous loop.
-                    local isArrayKey = type(k) == "number" and k >= 1 and k <= arrayCount and not IsFractional(k)
-                    if not isArrayKey and (entireMapSerializable or self:_ShouldSerialize(tab, k, v, opts, filter)) then
+                if opts.stable then
+                    local mapKeys = {}
+                    for k in pairs(tab) do
```

Comment:
You left `v` off of this, but the call to `_ShouldSerialize` still uses it (as it should). Use something like https://www.townlong-yak.com/globe/ or https://www.curseforge.com/wow/addons/findglobals to check for any other errors like this.